### PR TITLE
17.16 Collection Settings Update

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates documentation for the new Collections Settings screen.

- Removed most individual collection settings pages, replaced them with one View Collection Settings page.
- Removed unused files and images.
- Updated Data tab documentation to reflect that Enable Stream is no longer present.
- Added images to Truncate Collections and Delete Collections.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
Reviewers, please review.

File: docs/collections/view-collection-settings.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- **Stream Enabled** - Click to turn streams on or off for this collection. When streams are enabled, you can access the Stream tab.
+- **Stream Name** - The name of the collection stream.
+- **Replication** - Whether the stream is replicated globally or locally.
+- **WebSocket URL** - The WebSocket connection to the collection stream allows you to interact with the stream in real-time. This connection allows you to receive live updates as documents are added, updated, or removed from the collection.
+
+![Collection Stream Section](/img/collections/collection-stream.png)
+
+**Connections and Reload Collections**
+
+The Connections and Reload Collections sections only apply to [Document Store collections](./documents/). For more information about integrations and connections, refer to [Integrations](../integrations/).
```
Comment (grainier-macrometa) [inline docs/collections/view-collection-settings.md]: @diana-macrometa - Shall we mention somewhere about the behavior of the Reload Collections? It initially truncates the current collection and then only re-starts the Source connection.

**Review by grainier-macrometa — APPROVED** [review]

File: docs/collections/view-collection-settings.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/collections/view-collection-settings.md]: I did not know that. I was planning to put that sort of information in Integrations, but maybe it makes more sense here.

**Review by elof — APPROVED** [review]
:shipit:

**Pull request merged**